wordpress-mobile/gutenberg-mobile
# Fix crash when adding media objects.

Fixes #2367

GB PR: https://github.com/WordPress/gutenberg/pull/23058

To test:
 - Repeat the steps refered on the ticket and see that no crash longer happens.

PR submission checklist:
